drag_and_drop_v2/public/css/drag_and_drop_edit.css

```
@@ -1,10 +1,12 @@
 .xblock--drag-and-drop--editor {
   width: 100%;
   height: 100%;
+  margin-bottom: 0 !important;  /* Remove an undesired whitespace from Studio */
```

The following rule overrides it if `!important` was not used.

```
[class*="view-"] .modal-window .editor-with-buttons {
    margin-bottom: 60px;
}
```

The other alternative is do define it like below:

```suggestion
}

[class*="view-"] .modal-window .editor-with-buttons.xblock--drag-and-drop--editor {
 margin-bottom: 0;  /* Remove an undesired whitespace from Studio */
}
```

I'm personally fine with either approaches.